cmd/create/ingress/cmd.go

```diff
@@ -139,6 +158,41 @@ func run(cmd *cobra.Command, _ []string) {
 	if len(routeSelectors) > 0 {
 		ingressBuilder = ingressBuilder.RouteSelectors(routeSelectors)
 	}
+
+	var lbType *string
+	if cmd.Flags().Changed("lb-type") {
+		lbType = &args.lbType
```

Comment:
Nitpick: Don't need to care if it is changed, just set it to args.lbType and if it is interactive it should go through the next part anyways.